src/components/Login/index.tsx

```diff
@@ -78,7 +78,11 @@ const Login: React.FC = () => {
         <LanguagePicker />
       </div>
       <div className="relative z-40 flex flex-col items-center px-4 mt-10 sm:mx-auto sm:w-full sm:max-w-md">
-        <img src="/logo_stacked.svg" className="max-w-full mb-10" alt="Logo" />
+        <img
+          src="/logo_stacked.svg"
+          className="max-w-full mx-auto mb-10"
```

Comment:
We are using flex to center here. Let's do the same in the other places? We shouldn't need mx-auto then.

Reply:
Gotcha!